### DISCUSSION TITLE
Single-end support for metaspades.py

### DISCUSSION THREAD

**Discussion by skchronicles** [ablab/spades#1026] [General]
Hello there,

I was just wondering what the status is on single-end support for `metaspades.py`. I, incorrectly, made the assumption that the `-s` option was for single-end datasets. It does not appear to the case.

I just tested it out with a single-end dataset, and it failed (which is to be expected based on your documentation):
![image](https://user-images.githubusercontent.com/18038345/189713693-fe39be60-c519-41e5-a9e4-e65894afa867.png)

![image](https://user-images.githubusercontent.com/18038345/189714167-46e5e33d-96fe-47e7-a1b1-a049cb88a0b5.png)

Please let me know if you still have plans to add single-end support in the near future, or if you are close to supporting this datatype.

Best Regards,
@skchronicles

**Comment by skchronicles** [comment]
Cc-ing @snurk for visibility

**Comment by asl** [comment]
Hello! There is no ETA on removing this restriction

**Comment by skchronicles** [comment]
Thank you for the update. I appreciate it!
